### ISSUE TITLE
"Use GitHub Copilot in Zed." popup freezes and does nothing

### ISSUE THREAD

**Issue by wzxu** [zed-industries/zed#9596]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

As title, when I hover the buttons they're highlighted, but clicking them does nothing. 
Copy doesn't copy. Connect to GitHub doesn't open browser. Cancel doesn't dismiss the popup.
I've tried deleting everything including Zed config as well as GitHub login from the keychain and it's still the same.
Can't get Copilot to work due to this.
<img width="519" alt="13461" src="https://github.com/zed-industries/zed/assets/5619348/9baed9c6-2948-4d57-b3b0-e94cf715f2b2">

### Environment

Zed: v0.128.0 (Zed Preview)
OS: macOS 14.4.0
Memory: 16 GiB
Architecture: aarch64

**Comment by Moshyfawn** [comment]
Have you noticed anything strange in the Zed logs when you try to login/activate Copilot zed: open log?

**Comment by tekacs** [comment]
Could this be because of the following code in this PR in the latest release?

```rust
            .capture_any_mouse_down(cx.listener(|this, _: &MouseDownEvent, cx| {
                cx.focus(&this.focus_handle);
                cx.stop_propagation();
            }))
```

From https://github.com/zed-industries/zed/commit/e10360713453d98a3a7befc54a0379b1757decb5

**Comment by Moshyfawn** [comment]
Sounds like it, as there are no events or errors in the logs.

**Comment by ConradIrwin** [comment]
Fixed in 0.128.1, thanks for the fast report!

**Comment by haeussma** [comment]
Have the described issue in 0.128.3.

```plaintext
2024-03-21T19:58:46+01:00 [WARN] Generic lsp request to node failed: Unhandled method textDocument/definition
2024-03-21T19:58:47+01:00 [INFO] Language server with id 1 sent unhandled notification client/unregisterCapability:
{
  "unregisterations": [
    {
      "id": "44a46941-a1ff-4c67-b044-81b6776343e9",
      "method": "workspace/didChangeWatchedFiles"
    }
  ]
}
2024-03-21T19:58:47+01:00 [INFO] Node runtime install_if_needed
2024-03-21T19:58:48+01:00 [INFO] starting language server. binary path: "/Users/max/Library/Application Support/Zed/node/node-v18.15.0-darwin-arm64/bin/node", working directory: "/", args: ["/Users/max/Library/Application Support/Zed/copilot/copilot-v0.5.0/dist/agent.js", "--stdio"]
2024-03-21T19:58:48+01:00 [INFO] Language server with id 0 sent unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-03-21T18:58:48.357Z] Agent service starting",
  "metadataStr": "[DEBUG] [agent] [2024-03-21T18:58:48.357Z]",
  "extra": [
    "Agent service starting"
  ]
}
2024-03-21T19:58:48+01:00 [INFO] Language server with id 0 sent unhandled notification client/registerCapability:
{
  "registrations": [
    {
      "id": "185fff1f-1d93-48f1-bd61-5259022160f3",
      "method": "workspace/didChangeWorkspaceFolders",
      "registerOptions": {}
    }
  ]
}
2024-03-21T19:58:48+01:00 [INFO] Language server with id 0 sent unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-03-21T18:58:48.365Z] Telemetry initialized",
  "metadataStr": "[DEBUG] [agent] [2024-03-21T18:58:48.365Z]",
  "extra": [
    "Telemetry initialized"
  ]
}
2024-03-21T19:59:09+01:00 [ERROR] Cancelled LSP request task for "signInConfirm" id 9 which took over 120s
2024-03-21T19:59:09+01:00 [ERROR] crates/lsp/src/lsp.rs:720: oneshot canceled
2024-03-21T20:01:06+01:00 [ERROR] Cancelled LSP request task for "signInConfirm" id 5 which took over 120s
2024-03-21T20:01:15+01:00 [ERROR] Cancelled LSP request task for "signInConfirm" id 7 which took over 120s
```

**Comment by Moshyfawn** [comment]
@haeussma, would you mind double checking the version you're running by selecting `zed: about` in the Command Palette? The latest version is `0.128.1` and the last version with patch number `3` was `0.127.3`.

**Comment by Moshyfawn** [comment]
The fix was also brought into Stable in [v0.127.4](https://zed.dev/releases/stable/0.127.4).
